tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -1198,10 +1198,25 @@ static uint8_t start_discovery(const void *cmd, uint16_t cmd_len,
 		return br_start_discovery(cp);
 	}
 
-	/* Start LE scanning */
-	if (bt_le_scan_start(cp->flags & BTP_GAP_DISCOVERY_FLAG_LE_ACTIVE_SCAN ?
-			     BT_LE_SCAN_ACTIVE : BT_LE_SCAN_PASSIVE,
-			     device_found) < 0) {
+	struct bt_le_scan_param scan_param = {
```

Comment:
Should this use the `BT_LE_SCAN_PARAM_INIT()` helper macro instead of manually building the struct? Not sure how this is typically done in the BT code.